devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -67,9 +67,15 @@ metadata {
 		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 1) {
 			state "default", label:"", action:"refresh.refresh", icon:"st.secondary.refresh"
 		}
+		valueTile("shadeLevel", "device.level", width: 4, height: 1) {
+			state "level", label: 'Shade is ${currentValue}% up', defaultState: true
+		}
 		valueTile("batteryLevel", "device.battery", width: 2, height: 2) {
 			state "battery", label:'${currentValue}% battery', unit:""
 		}
+		controlTile("levelSliderControl", "device.level", "slider", width:2, height: 1, inactiveLabel: false) {
```

Comment:
While the tiles are not used any more, for the sake of cleanliness, let's please not add this and above to the PR.